lib/galaxy/tools/tag_collection_from_file.xml

```diff
@@ -1,8 +1,8 @@
 <tool id="__TAG_FROM_FILE__"
-      name="Tag elements from file"
+      name="Tag collection elements"
       version="1.0.0"
       tool_type="tag_from_file">
-    <description>from contents of a file</description>
```

Comment:
One could imagine other ways to tag elements - I mean we should have tools that apply simple rules to do this and another that takes an input collection (perhaps from a downstream analysis) and a collection to mirror (perhaps an input) and re-tags the input based on the mirror. For these reasons I think placing file in the description makes a lot of sense - but I won't die on this hill if conciseness is important here I can respect that.

Reply:
yes, let's return to this once we have more tools :)